### DISCUSSION TITLE
Different syntax highlight in dev and prod build

### DISCUSSION THREAD

**Discussion by OCram85** [thuliteio/doks#520] [General]
Hi @h-enk,

just noticed there the syntax highlight differs between dev and an prod builds.

In my case I switched to monokai theme an got different results:

- local dev build: `npm run server`

![image](https://user-images.githubusercontent.com/13993700/137860464-11e2c404-35ff-4457-b4c6-b1d5114a601f.png)

- prod build served by caddy server produced by `npm run build` and packed into docker container:
 
![image](https://user-images.githubusercontent.com/13993700/137860485-00c02d73-bdb2-464c-9d0c-9aab239c4644.png)

:bulb: This happens in light & dark mode

Do you have any ideas what could cause this? Is there a different loading order for css / js files ?

**Comment by OCram85** [comment]
So it seems like the `npm run server` builds adds aditional scss:

![image](https://user-images.githubusercontent.com/13993700/137861141-458d9831-2848-4adb-9ab1-12ef6fa80da2.png)

![image](https://user-images.githubusercontent.com/13993700/137861329-ec8bcd86-7bde-49dc-831a-2e0a9727e68c.png)

**Comment by h-enk** [comment]
If there's a difference between the dev and prod builds, it'll probably have something to do with PurgeCSS.

In `./config/postcss.config.js`, section `...whitelister([`, add `'./node_modules/highlight.js/scss/monokai.scss',`, and try running the prod build again.

**Reply by OCram85** [reply]
Awesome, you're my hero!!! - Just works perfect now...

![image](https://user-images.githubusercontent.com/13993700/137864003-b32f4d1a-3e22-4392-b9f4-08d541a1a21d.png)

**Reply by h-enk** [reply]
Haha, good to hear!
